third_party/blink/renderer/core/html/media/html_media_element.cc

```diff
@@ -570,6 +573,10 @@ HTMLMediaElement::HTMLMediaElement(const QualifiedName& tag_name,
   AddElementToDocumentMap(this, &document);
 
   UseCounter::Count(document, WebFeature::kHTMLMediaElement);
+
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  RuntimeEnabledFeatures::SetH5vccAudioConnectorsEnabled(true);
```

Comment:
We should try to enable this feature via command line in CobaltActivity.java (for ATV) via `--enable-blink-features`.

On Linux, feel free to put a TODO or put the corresponding place in the code.